src/uwtools/drivers/mpas.py

```diff
@@ -0,0 +1,167 @@
+"""
+A driver for the MPAS component.
+"""
+
+from datetime import datetime, timedelta
+from pathlib import Path
+from typing import Optional
+
+from iotaa import asset, dryrun, task, tasks
+
+from uwtools.api.template import render
+from uwtools.config.formats.nml import NMLConfig
+from uwtools.drivers.driver import Driver
+from uwtools.strings import STR
+from uwtools.utils.tasks import file, filecopy, symlink
+
+
+class MPAS(Driver):
+    """
+    A driver for MPAS.
+    """
+
+    def __init__(
+        self,
+        cycle: datetime,
+        config: Optional[Path] = None,
+        dry_run: bool = False,
+        batch: bool = False,
+    ):
+        """
+        The driver.
+
+        :param cycle: The cycle.
+        :param config: Path to config file (read stdin if missing or None).
+        :param dry_run: Run in dry-run mode?
+        :param batch: Run component via the batch system?
+        """
+        super().__init__(config=config, dry_run=dry_run, batch=batch, cycle=cycle)
+        if self._dry_run:
+            dryrun()
+        self._cycle = cycle
+
+    # Workflow tasks
+
+    @task
+    def boundary_files(self):
+        """
+        Boundary condition files.
+        """
+        yield self._taskname("boundary files")
```

Comment:
Slight disagreement between method name, docstring (which appears in CLI `--help`) and task name as yielded to `iotaa`. Maybe remove `condition` from the docstring for consistency? Whatever you think is best, though.

Reply:
Same complain for the init version.